package.json

```diff
@@ -14,7 +14,7 @@
     "release": "yarn test && yarn publish --conventional-commits --changelog-preset=angular"
   },
   "peerDependencies": {
-    "botfuel-dialog": "5.x"
+    "botfuel-dialog": "7.x"
```

Comment:
Do we provide a version for users of botfuel-dialog 6.x?

Reply:
Ah, right.  So I should deploy the module two times, the first time with the dependency: ">=5 <7" and the second time with the dependency "7.x"?

Reply:
It seems to me that in this PR you fixes changes that were introduced by 6 *and* 7. I would split that work into 2 PRs, one for 5->6 and 6->7.
This in not a lot of work using git tools like kraken. Ask me if you want.

Reply:
does it neccessary?